appeals/api/src/server/common/validators/string-parameter.js

```diff
@@ -31,7 +31,7 @@ const validateStringParameter = (parameterName, maxLength = LENGTH_300) =>
  */
 export const validateStringParameterAllowingEmpty = (parameterName, maxLength = LENGTH_300) =>
 	body(parameterName)
-		.optional()
+		.optional({ nullable: true })
```

Comment:
not sure if this was necessary ?

Reply:
It was unfortunately as it was submitting a null which optional doesn't accept by default.